pkg/apis/core/types.go

```diff
@@ -3595,6 +3595,13 @@ type PodStatus struct {
 	// +featureGate=InPlacePodVerticalScaling
 	// +optional
 	Resize PodResizeStatus
+
+	// TotalResourcesRequested is the effective resource requests for this pod taking into consideration init
```

Comment:
this will include the requested for InPlace resize, correct? Maybe we can mention it here?